app/components/notification_message_component.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="govuk-notification-banner govuk-notification-banner--<%= notification_type %>" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+  <div class="govuk-notification-banner__header">
+    <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+      <%= title %>
+    </h2>
+  </div>
+  <div class="govuk-notification-banner__content">
+    <p class="govuk-notification-banner__heading">
+      <%= heading %>
+    </p>
+    <p class="govuk-body">
+      <% if message.present? %>
```

Comment:
if we don't have a message/link we are rendering the following `<p class="govuk-body"></p>`, maybe we should move this check above the section to avoid rendering an empty section?